### DISCUSSION TITLE
Incredible work

### DISCUSSION THREAD

**Discussion by aceslick911** [gdombiak/OctoPod#639] [General]
Just wanted to let all devs and contributors know that I absolutely love this app (iOS) and it’s seriously my favourite one. 

Thanks for all your effort! Will try to do more than say thanks soon 🤔

**Comment by gdombiak** [comment]
Thanks @aceslick911 Angelo for the kind words!! I try to incorporate as much as possible from the feedback given by the community in the my little available free time. If you have ideas of new features or improvements please let me know.

Thanks,
Gaston
